deployments/gpu-operator/templates/serviceaccount.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: special-resource-operator
-  namespace: gpu-operator
+  namespace: {{ $.Release.Namespace }}
```

Comment:
Just to be clear, this means the GPU Operator is by default installed in the default namespace unless the user specifies a namespace and uses `--create-namespace`

Reply:
its released in current or specified `--namespace`.
It's the best practice for helm charts

Reply:
Ok, I'll update the readme then